draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -186,7 +186,7 @@ Nevertheless, specifying congestion control algorithms has a number of advantage
   which can make it easier for them to suggest improvements and/or identify
   limitations. Further, the specification can help multiple contributors align
   on a consensus change to the algorithm.
-- A specification that is accessible to anyone circumvents the issue that some
+- A specification that is accessible to anyone, can circumvent the issue that some
```

Comment:
The comma here looks superfluous?

AFAIK, in a statement of the form "An X that is Y can Z" there should not be a comma after the Y...